app/controllers/static_controller.rb

```diff
@@ -1,5 +1,11 @@
 class StaticController < ApplicationController
   def index
+    @press = {
+      'fast-company' => 'http://fastcompany.com',
+      'forbes' => 'http://forbes.com',
+      'huffington-post' => 'http://www.huffingtonpost.com/dr-shelley-prevost/creating-a-better-world-o_b_6061626.html',
+      'tedxteen' => 'http://www.tedxteen.com/speakers-performers/tedxteen-2014-london/237-ankit-shah'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.